mscr-ui/src/modules/form/index.tsx

```diff
@@ -108,12 +111,12 @@ export default function FormModal({
     usePutSchemaMscrCopyMutation();
   const [submitAnimationVisible, setSubmitAnimationVisible] =
     useState<boolean>(false);
-
+  
   const formDataFromInitialData = useCallback(() => {
     if (!initialData) return;
     const existingData: FormType = {
       format:
-        modalType == ModalType.MscrCopy ? Format.Mscr : initialData.format,
+        modalType == ModalType.MscrCopy? Format.Mscr : initialData.format,
```

Comment:
There should be a space between the conditional operators and the operands, so restore space between 'ModalType.MscrCopy' and '?'